ml/include/pcl/ml/dt/decision_tree_trainer.h

```diff
@@ -229,11 +229,11 @@ class PCL_EXPORTS DecisionTreeTrainer {
 
 private:
   /** Maximum depth of the learned tree. */
-  std::size_t max_tree_depth_;
+  std::size_t max_tree_depth_{15};
   /** Number of features used to find optimal decision features. */
-  std::size_t num_of_features_;
+  std::size_t num_of_features_{1000};
   /** Number of thresholds. */
-  std::size_t num_of_thresholds_;
+  std::size_t num_of_thresholds_{10};
 
   /** FeatureHandler instance, responsible for creating and evaluating features. */
   pcl::FeatureHandler<FeatureType, DataSet, ExampleIndex>* feature_handler_;
```

Comment:
```suggestion
  pcl::FeatureHandler<FeatureType, DataSet, ExampleIndex>* feature_handler_{nullptr}; 
```
And removed from the constructor initializer list?

Reply:
There are more fields that can be done the same with, ie. stats_estimator_?

I'm not sure about the vector types, if they can be with just {} to create the empty vectors? ie. label_data_, examples_ and the data_set_ (which is of DataSet type)